src/ntapfuse_ops.c

```diff
@@ -0,0 +1,362 @@
+/**
+ * Project: ntapfuse
+ * Author: Samuel Kenney <samuel.kenney48@gmail.com>
```

Comment:
Add your name to the author's list name as you are making changes in this file

Reply:
Just added